src/types/Chat.ts

```diff
@@ -0,0 +1,16 @@
+export enum CHAT_TYPES {
```

Comment:
```suggestion
export enum ChatTypes {
```

I would potentially also suggest that each of the entries also be named similarly, eg. `WorldChat` rather than `WORLD_CHAT`